src/libraries/Microsoft.PowerFx.Core/Texl/BuiltinFunctionsCore.cs

```diff
@@ -209,7 +209,8 @@ internal class BuiltinFunctionsCore
         public static readonly TexlFunction StdevPT = _library.Add(new StdevPTableFunction());
         public static readonly TexlFunction Substitute = _library.Add(new SubstituteFunction());
         public static readonly TexlFunction SubstituteT = _library.Add(new SubstituteTFunction());
-        public static readonly TexlFunction Sum = _library.Add(new SumFunction());
+        public static readonly TexlFunction Sum = _library.Add(new SumFunction());
+        public static readonly TexlFunction Summarize = _library.Add(new SummarizeFunction());
         public static readonly TexlFunction SumT = _library.Add(new SumTableFunction());
```

Comment:
This should not be added to the core library directly until Power Apps is ready to consume it. Please add it to _featureGateFunctions instead.